util/tlgen/elaborate.py

```diff
@@ -150,39 +157,49 @@ def process_pipeline(xbar):
         if dnode.node_type == NodeType.ASYNC_FIFO:
             continue
 
+        req_pass = 1 if host.req_fifo_pass else 0
+        rsp_pass = 1 if host.rsp_fifo_pass else 0
         if dnode.node_type == NodeType.SOCKET_1N:
             if full_fifo:
-                dnode.hpass = 0
+                dnode.hreq_pass = 0
+                dnode.hrsp_pass = 0
                 dnode.hdepth = 2
             elif fifo_passthru:
-                dnode.hpass = 0
+                dnode.hreq_pass = req_pass
+                dnode.hrsp_pass = rsp_pass
                 dnode.hdepth = 2
             elif full_passthru:
-                dnode.hpass = 1
+                dnode.hreq_pass = 1
+                dnode.hrsp_pass = 1
                 dnode.hdepth = 0
 
-            log.info(
-                "Finished processing socket1n {}, pass={}, depth={}".format(
-                    dnode.name, dnode.hpass, dnode.hdepth))
+            print(
```

Comment:
could you revert these back? I think if you add `--verbose` then you can see the INFO logs.

Reply:
ah yeah sorry, this one is meant to be rebased away. If you look at #11624 you'll see that I reverted already.

Reply:
alright! i rebased.